org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -623,6 +672,33 @@ private float IntersectQuad(Vector3 start, Vector3 end, float depth)
             }
         }
 
+        /// <summary>
+        /// Set the enabled state of the Front plate Raw Image if any.
+        /// </summary>
+        /// <param name="enable">True to enable, false to disable</param>
+        protected void SetEnableFrontPlateRawImage(bool enable)
+        {
+            GameObject frontPlateGameObject = transform.Find(FRONT_PLATE_NAME).gameObject;
+            UnityEngine.UI.RawImage frontPlateRawImage = frontPlateGameObject?.GetComponent<UnityEngine.UI.RawImage>();
```

Comment:
Can we cache these image references on startup instead of calling `Find` and `GetComponent` on every call?

Reply:
I initially thought on something similar but then I realized that such change would also require to implement a way to update the cached image reference if someone develops an app in which the Front Plate Raw Images are updated in runtime.  Then I thought that a Find + GetComponent call would increase the computational complexity
with a constant O(k) which is acceptable if it doesn't happen often (it doesn't happen on a per-frame basis).  So, considering it doesn't impact performance significantly and aiming to keep the code change simple I thought on implementing it as I initially suggested.  May I know your thoughts on my reasoning?  Also, if your company has a specific need to have it cached that we are not aware of then please let us know.

Reply:
I really don't like referencing objects by name (= the use of transform.Find() or GameObject.Find()), even when cached.
What if a user when to rename the frontplate gameobject?

Reply:
An alternative may be to use a serialized field (like how InteractablePulse stores the front plate).

Reply:
Thank you all for your feedback, I've tried to capture it in the latest commit.